.changeset/fuzzy-houses-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/picker": patch
+---
+
+chore: upgrade picker border-color
```

Comment:
This needs to be more detailed about what properties were specifically updated and what the impact might be for consumers.